public/app/plugins/datasource/tempo/plugin.json

```diff
@@ -27,6 +27,14 @@
       {
         "name": "GitHub Project",
         "url": "https://github.com/grafana/tempo"
+      },
+      {
+        "name": "Raise issue",
```

Comment:
we should check this given the guidance from Loki team was to still raise in grafana

Reply:
Yeah, change it to grafana/grafana please